File: message-bus-demos/demos-common/src/main/java/alma/adapt/examples/common/Person.java
package alma.adapt.examples.common;

import alma.icd.adapt.messagebus.AbstractMessage;

public class Person extends AbstractMessage {
	
	public String name;
	public int age;
	public boolean alive;

	public Person() {
	}
	
	public Person( String name, int age, boolean alive ) {
		this.age = age;
		this.alive = alive;
		this.name = name;
	}

	@Override
	public String toString() {
		return this.getClass().getSimpleName()
				+ "[name=" + name + ", age=" + age + ", alive=" + alive + "]";
	}
}

File: message-bus-demos/demos-common/src/main/python/alma/adapt/examples/common/Person.py
from adapt.messagebus.AbstractMessage import AbstractMessage

class Person(AbstractMessage):
    def __init__(self, name=None, age=None, alive=None):
        self.name = name
        self.age = age
        self.alive = alive
    def __str__(self):
        return self.__class__.__qualname__ + "[name=" + self.name + ", age=" + str(self.age) + ", alive=" + str(self.alive) + "]"
